sixtyfps_compiler/lib.rs

```diff
@@ -122,6 +122,21 @@ pub async fn compile_syntax_node(
 
     let mut loader =
         typeloader::TypeLoader::new(global_type_registry, &compiler_config, &mut diagnostics);
+
+    let known_styles = fileaccess::styles();
```

Comment:
Can you actually move this code to `TypeLoader::new` so that the other style-related warning is together.
The alternative would be to move the style check from `TypeLoader::new` to here. But then we need to be careful about about the LSP that doesn't use this code path

Reply:
Implemented.